apps/api/src/server/repositories/representation.repository.js

```diff
@@ -480,6 +480,36 @@ export const deleteApplicationRepresentationAttachment = async (attachmentId) =>
 	});
 };
 
+/**
+ *
+ * @param {number} repId
+ * @param {object}action
+ * @returns {Promise<*>}
+ */
+export const updateApplicationRepresentationStatus = async (repId, action) => {
+	const representation = await databaseConnector.representation.findFirst({ where: { id: repId } });
+
+	const updateRepStatus = databaseConnector.representation.update({
```

Comment:
Do we want to check that representation was found first?

Reply:
thats done in the middleware check.